index.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/index.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,500;0,600;1,500&display=swap"
+      rel="stylesheet"
+    />
+    <title>BEM Practice</title>
+  </head>
+  <body>
+    <!-- Header -->
+    <section class="home">
+      <div class="header">
+        <div class="header__menu">
+          <h1 class="header__men__logo">Medical Core</h1>
+          <ul class="header__nav__list">
+            <li class="header__nav__list__item header__nav__list item--active">
```

Comment:
you have two block elements, are you accessing each ones style? if not i think having one class name and a modifier is best.